app/static/js/index.js

```diff
@@ -347,3 +347,24 @@ window.onscroll = function(e) {
         pagesFinished = true;
     }
 }
+
+// Scroll to top button
+var scrollTopBtn = document.getElementById('scrollToTopBtn')
+var rootElement = document.documentElement
+
+function showButton() {
+if ((rootElement.scrollTop / (rootElement.scrollHeight - rootElement.clientHeight)) > 0.15 ) {
+    scrollTopBtn.classList.add('showBtn')
```

Comment:
Rename this class to `shown` and target it in CSS with `#scroll_top_button.shown`; we know it's a button already so you can keep the class name minimal